### DISCUSSION TITLE
续费1年的蓝灯专业版，但邮箱无法收到验证码登录

### DISCUSSION THREAD

**Discussion by gggg0034** [getlantern/lantern-forum-cn#785] [Q&A]
您好，我支付了一年的蓝灯专业版，附件里是我的支付记录。
我的蓝灯账号是:735945640@qq.com
（如果上述邮箱对贵团队信息接收困难，也可以发送邮箱到我的谷歌邮箱：zhiwenzhu1998@gmail.com）
期待贵团队的回复！
![zzw20240815V2](https://github.com/user-attachments/assets/cd2a91a8-770a-4821-bcf4-952958a5c083)

**Comment by MaxMKMax** [comment]
@gggg0034 您好. 已经帮您提交问题上去了, 请放心我们会帮您解决问题的. 谢谢您支持蓝灯!
